<launch>


<!-- Segmentation -->


<node name="cloud_file_segmentation" pkg="kinect_scan" type="cloud_file_segmentation" output="screen" >
	

	<param name="input_cloud_file" type="string" value="/home/hugo/ros/src/kinect_scan/data/classroom.pcd" />
	<param name="output_cloud_file" type="string" value="/home/hugo/ros/src/kinect_scan/data/data.ply" />


	<remap from="/cloud_file_segmentation/output" to="/cloud_file_segmentation/pointCloud_total_segmented"/>

	<remap from="/cloud_file_segmentation/output1" to="/cloud_file_segmentation/pointCloud_segmented1"/>
	<remap from="/cloud_file_segmentation/output2" to="/cloud_file_segmentation/pointCloud_segmented2"/>
	<remap from="/cloud_file_segmentation/output3" to="/cloud_file_segmentation/pointCloud_segmented3"/>
	<remap from="/cloud_file_segmentation/output4" to="/cloud_file_segmentation/pointCloud_segmented4"/>
	<remap from="/cloud_file_segmentation/output5" to="/cloud_file_segmentation/pointCloud_segmented5"/>
	<remap from="/cloud_file_segmentation/output6" to="/cloud_file_segmentation/pointCloud_segmented6"/>
	<remap from="/cloud_file_segmentation/output7" to="/cloud_file_segmentation/pointCloud_segmented7"/>
	<remap from="/cloud_file_segmentation/output8" to="/cloud_file_segmentation/pointCloud_segmented8"/>
	<remap from="/cloud_file_segmentation/output9" to="/cloud_file_segmentation/pointCloud_segmented9"/>
	<remap from="/cloud_file_segmentation/output10" to="/cloud_file_segmentation/pointCloud_segmented10"/>
	<remap from="/cloud_file_segmentation/output11" to="/cloud_file_segmentation/pointCloud_segmented11"/>


	<param name="ClusterTolerance" type="double" value="0.1" />
	<param name="MinClusterSize" type="int" value="150" />
	<param name="DistanceThreshold" type="double" value="0.005" />
	<param name="Leafsize" type="double" value="0.05" />

	<param name="ZPassThrough" type="bool" value="true" />
	<param name="zMin" type="double" value="-0.50" />
	<param name="zMax" type="double" value="1.5" />

	<param name="show_toolbar" type="bool" value="true" />


</node>


</launch>
